.github/workflows/prover-test.yaml

```diff
@@ -1,13 +1,15 @@
 name: "Prover Test"
 on:
-  pull_request:
-  push:
-    branches:
-      - main
-      - devnet
-      - testnet
-      - auto
-      - canary
+  # Stop running on PR / main until this is fixed
```

Comment:
cc @junkil-park @wrwg @meng-xu-cs to uncomment when the prover test is ready again